enterprise/reporting/reporting.go

```diff
@@ -295,7 +295,7 @@ func (handle *HandleT) getAggregatedReports(reports []*types.ReportByStatus) []*
 
 func (handle *HandleT) mainLoop(ctx context.Context, clientName string) {
 	tr := &http.Transport{}
-	netClient := &http.Client{Transport: tr, Timeout: config.GetDuration("HttpClient.timeout", 30, time.Second)}
+	netClient := &http.Client{Transport: tr, Timeout: config.GetDuration("HttpClient.timeout", 60, time.Second)}
```

Comment:
we are using this config property in other places too. Should we provide a special config option for reporting?

Reply:
adding to @atzoum  comment. We might want to modify `GetDuration` func to take multiple keys, just like we do for  `RegisterDurationConfigVariable` func.
here: https://github.com/rudderlabs/rudder-server/blob/06440975a2be136d77ab09928c03e5638481f379/config/config.go#L483

or maybe create a new func that takes multiple keys.